smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/http-uri-conflict-validator.json

```diff
@@ -95,6 +122,167 @@
                     "method": "GET"
                 }
             }
+        },
+        "ns.foo#H": {
```

Comment:
Can there be a test case that conflicts with a label that doesn't have a pattern at all to verify that's handled correctly?